.github/workflows/main2.yml

```
@@ -0,0 +1,5 @@
+- name: Send PR Review to External App
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)